modules/bm_apps/examples/Makefile.am

```diff
@@ -1,4 +1,14 @@
 AM_CPPFLAGS = -I$(top_srcdir)/modules/bm_apps/include
+
+if P4THRIFT
+  AM_CPPFLAGS+=$(THRIFT_INC) $(THRIFT_LIB)
```

Comment:
Just trying to link the correct library and this worked. Is there a better variable I should use?